git_machete/cli.py

```diff
@@ -619,7 +619,8 @@ def launch(orig_args: List[str]) -> None:
             git.expect_no_operation_in_progress()
             current_branch = git.get_current_branch()
             dest = machete_client.parse_direction(
-                parsed_cli.direction, current_branch, allow_current=False, down_pick_mode=True)[0]  # with down_pick_mode=True there is only one element in list allowed
+                parsed_cli.direction, current_branch, allow_current=False, down_pick_mode=True)[
+                0]  # with down_pick_mode=True there is only one element in list allowed
```

Comment:
Huh that look very awkward... maybe just merge code from lines 621-623 to a single line, and move comment to line above?